templates/form/layout.html.twig

```diff
@@ -1,16 +1,18 @@
-{% extends 'bootstrap_3_layout.html.twig' %}
+{% extends 'bootstrap_4_layout.html.twig' %}
 
 {# Errors #}
 
+{#
 {% block form_errors -%}
     {% if errors|length > 0 -%}
         {% if form is not rootform %}<span class="help-block">{% else %}<div class="alert alert-danger">{% endif %}
         <ul class="list-unstyled">
         {%- for error in errors -%}
-            {# use font-awesome icon library #}
+            {# use font-awesome icon library # }
```

Comment:
to rollback

Reply:
Ok i rolled it back and i got this
<img width="883" alt="Screen Shot 2020-12-30 at 1 21 09 AM" src="https://user-images.githubusercontent.com/13308700/103324013-6ac9c280-4a3d-11eb-8033-a9b8eecb4426.png">

Reply:
And the test `testAdminNewDuplicatedPost` method is failing because we no longer have `form-group.has-error`

Reply:
i was commenting to rollback the extra space on the precise line i commented on

from

`{# use font-awesome icon library #}`

to
 `{# use font-awesome icon library # }`

sorry if it was unclear :s

Reply:
Ah sorry.
The reason why I added that extra space is when I don't do so the code after that line will not be commented